### DISCUSSION TITLE
Multicurrency in Pimcore Ecommerce

### DISCUSSION THREAD

**Discussion by vineethvenugopal5** [pimcore/pimcore#7875] [General]
How can i activate multi currency in pimcore ? Should i need to use Coreshop for multi currency ?

**Comment by alasdair-shields** [comment]
It is possible without Coreshop (might be easier with Coreshop but I haven't tried).

This method uses a currency conversion table to convert from that entered in Pimcore admin to the user's currency.

I created a new PriceSystem that extends Pimcore\Bundle\EcommerceFrameworkBundle\PriceSystem\AttributePriceSystem

I then over-rode createPriceInfoInstance to convert the amount entered for the object in Pimcore admin (GBP in my case) to the user's local currency.  I also created a currency converter utility and injected it into the AttributePriceSystem constructor which I then used to perform the actual conversion.

**Reply by vineethvenugopal5** [reply]
@alasdair-shields - Thank you very much. I will try it out.

**Reply by dpfaffenbauer** [reply]
CoreShop has conversion by default. CoreShop also saves the Cart and Order in both currencies: Store Currency and User Currency, to have the proper information for whoever needs it.

**Reply by vineethvenugopal5** [reply]
@dpfaffenbauer - I have installed Coreshop pimcore bundle , but after that when i am installing corshop resource bundle its getting an error message like this "You have requested a non-existent parameter "coreshop.resources" ". Do we need to install the coreshop package fully to avail the multi currency ?
